invoke-ai/InvokeAI
# Report ci disk space + minor docker fixes

## What type of PR is this? (check all applicable)



## Have you discussed this change with the InvokeAI team?


## Description

This PR 
- adds logging of free disk space on the CI runner before and after the disk space cleanup step, to understand how much space we're actually saving and whether the cleanup is sufficient to support the container build. This doesn't fix anything but rather informs our next steps on it.
- adds a 40-minute timeout to the container build job so that the job isn't clogging up CI for 6 hours if it's unable to proceed
- updates the UI builder image to nodejs 20.x
- removes linting step from the UI build in container and runs the build job directly. Linting currently fails for seemingly legitimate reasons, but this should be fixed separately and should not block the build.
- adds `.pnpm-store` directories to `.gitignore`
- proposes adding @ebr to CODEOWNERS for `docker` and `workflows`

## QA Instructions, Screenshots, Recordings

- test this by building and running the container locally: `cd docker; docker compose build; docker compose up`. The container should build  without errors and the application should function as normal
- with repo admin access, test this in CI by kicking off a container build from this branch


## Merge Plan

This PR can be merged when approved
